runtime/t3rn-parachain/src/lib.rs

```diff
@@ -38,24 +33,19 @@ use frame_support::{
     },
 };
 use frame_system::limits::{BlockLength, BlockWeights};
-pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
-pub use sp_runtime::{MultiAddress, Perbill, Permill};
-
-#[cfg(any(feature = "std", test))]
-pub use sp_runtime::BuildStorage;
-
-// Polkadot Imports
 use polkadot_runtime_common::BlockHashCount;
 use polkadot_runtime_constants::weights::RocksDbWeight;
+use sp_api::impl_runtime_apis;
+use sp_core::{crypto::KeyTypeId, OpaqueMetadata};
+use sp_runtime::{
+    create_runtime_str, generic,
+    traits::{AccountIdLookup, Block as BlockT},
+    transaction_validity::{TransactionSource, TransactionValidity},
+    ApplyExtrinsicResult,
+};
+use sp_std::{convert::TryInto, prelude::*};
 
-/// Executive: handles dispatch to the various modules.
-pub type Executive = frame_executive::Executive<
-    Runtime,
-    Block,
-    frame_system::ChainContext<Runtime>,
-    Runtime,
-    AllPalletsWithSystem,
->;
+pub const EXISTENTIAL_DEPOSIT: u128 = 500;
```

Comment:
We could simplify this by setting it to 1. Would allow most users to forget about the existential deposit and probably only be relevant in situations where we do passive payouts. Cannot be zero though because it is used as divisor somewhere in the runtime.

Reply:
Agreed